### PR TITLE
LKE-5266: authentication required by default

### PR DESCRIPTION
**Pull request by gmasclet**
https://linkurious.atlassian.net/browse/LKE-5266

_As an Admin, I expect that only authorized users have access to my instance of LKE._

### CODE REVIEW

File: src/api/Auth/index.ts [hunk 1]
```diff
@@ -78,6 +83,22 @@ export class AuthAPI extends Request {
     return response;
   }
 
+  /**
+   * Create the current user (in order to setup authentication).
+   */
+  public async createCurrentUser(this: Request<User>, params: ICreateCurrentUserParams) {
```
Comment (aminerhanemi) [inline src/api/Auth/index.ts:89]: can we name it createFirstUser ? just a suggestion

**Review by aminerhanemi — CHANGES_REQUESTED** [review]

File: src/api/Auth/types.ts [hunk 2]
```diff
@@ -15,6 +15,12 @@ export interface ILoginOAuth2Params {
   state: string;
 }
 
+export interface ICreateCurrentUserParams {
```
Comment (aminerhanemi) [inline src/api/Auth/types.ts:18]: same

**Comment by aminerhanemi** [comment]
I added the ci:upload label so we can test it with the server and frontend

File: src/api/Auth/index.ts (hunk 1 above)
Comment (gmasclet) [reply src/api/Auth/index.ts]: I renamed it `setupAuthentication()` as agreed during our Zoom call.

File: src/api/Auth/types.ts (hunk 2 above)
Comment (gmasclet) [reply src/api/Auth/types.ts]: Renamed here too :ok_hand:

**Pull request merged**